airflow/models/taskinstance.py

```diff
@@ -1517,6 +1534,7 @@ def xcom_pull(
             task_ids=None,
             dag_id=None,
             key=XCOM_RETURN_KEY,
+            execution_date=None,
```

Comment:
wait we didn't get this part in? 😮

Reply:
oh wait... the default they have is fine. actually, we could remove the changes to this function. i think it was there before to mostly to just default it to self.execution date which it does now. 

```
$ git grep xcom_pull | grep -c 'execution_date'
0
```